src/stream/src/executor/over_window/general.rs

```diff
@@ -382,7 +387,15 @@ impl<S: StateStore> OverWindowExecutor<S> {
                                     yield chunk;
                                 }
                             }
-                            _ => panic!("other cases should not exist"),
+                            _ => {
+                                let vnode = this.state_table.compute_vnode_by_pk(&key.pk);
+                                let raw_key = serialize_pk_with_vnode(
+                                    &key.pk,
+                                    this.state_table.pk_serde(),
+                                    vnode,
+                                );
+                                panic!("other cases should not exist. raw_key: {:?}, existed: {:?}, new: {:?}", raw_key, tp1, tp2);
```

Comment:
Feeling a little bit weird to print raw table key in executor. But it looks acceptable for error avoidance. I would prefer to move the `let tp1 = existed.to_record_type(); ...` part and the `use` statement to the panic branch to make the debug things stay together.

Reply:
fix